classfile.h

```diff
@@ -51,6 +51,7 @@ typedef struct {
     char *class_name;
     char *name;
     char *descriptor;
+    variable_t *variable; /* store static fields in the class */
```

Comment:
Is `variable` a reasonably explicit name for its purpose?

Reply:
It uses to store static variable. Is it better to use `static_var` or `static_field` instead?

Reply:
`static_var` sounds good.